api/report_storage_server.thrift

```diff
@@ -52,16 +52,8 @@ service CheckerReport {
 
                 // The next few following functions must be called via the same connection.
```

Comment:
Is this still valid?

I mean do we really want to handle storage transactions (storage of reports belonging to the same run instance) based on connections? What if a connection is breaks down due to network error? Will the run under being updated be broken too?

Wouldn't it be more safe to handle transactions independently of connections?
Something like:

i64 **openStorageTransaction**(run_id) //returns a run storage transaction id
addReport(...)
addReport(...)
**closeStorageTransaction**(i64 storage_transaction_id)

when the closeTransaction is called, we set the status of the reports that were not present in the session would be set to resolved.